srs_data/sampling.py

```diff
@@ -52,32 +63,58 @@ def _validate_input(self):
         pass
 
     def _sample_production_logs(self):
+        hostname = ""
+        if self._client == 'condor':
+            hostname = 'comcast-kibana.asapp.com'
+        elif self._client == 'spear':
+            hostname = 'sprint-kibana.asapp.com'
+
         subprocess.run([
             sys.executable,
-            os.path.join(ASAPP_PRODML_ROOT, 'srs_data', 'harvest_cc_logs.py'),
+            os.path.join(ASAPP_PRODML_ROOT, 'tools', 'harvest_cc_logs.py'),
             '--dt_from', self._start_date + 'T0:0:0',
             '--dt_to', self._end_date + 'T0:0:0',
             '--output', os.path.join(self._output_directory, 'full-sample.csv'),
-            '--no-collapse'
+            '--no-collapse',
+            '--host', hostname
         ])
 
     def _take_uniform_sample(self):
-        subprocess.run([
-            sys.executable,
-            os.path.join(ASAPP_PRODML_ROOT, 'srs_data', 'hier_sample_logs.py'),
-            '--consolidate',
-            '--sample-size', '450',
-            '--custguid-blacklist', 'comcastblacklist:20170804',
-            'local://' + os.path.join(self._output_directory, 'full-sample.csv'),
-            self.uniform_sample_file
-        ])
+        if self._client == 'condor':
+            subprocess.run([
+                sys.executable,
+                os.path.join(ASAPP_PRODML_ROOT, 'tools', 'hier_sample_logs.py'),
+                '--consolidate',
+                '--sample-size', self._data_count,
+                '--custguid-blacklist', 'comcastblacklist:20170804',
```

Comment:
Not a blocker, but let's config this?

Reply:
doesn't make sense to config this because this should change really infrequently. I'd say try to minimize the config to things that would change on each run.

Reply:
Cool.